tests/test_bench.py

```
@@ -0,0 +1,923 @@
+        assert p["output"] == 180.00
+
+
+class TestRunJudgeDocstring:
+    """`run_judge.__doc__` must actually be the docstring — a triple-quoted
+    string placed after another statement in the body becomes an unused
+    runtime literal, not `__doc__`, and that broke `help(run_judge)`."""
+
+    def test_run_judge_has_docstring(self):
+        from evals.judges.pairwise_judge import run_judge
+        doc = run_judge.__doc__
+        assert doc is not None and doc.strip(), "run_judge has no docstring (or one placed after another statement)"
```

Ruff (0.15.14)

[warning] Assertion should be broken down into multiple parts

(PT018)